formats/json/commonMain/src/kotlinx/serialization/json/JsonElement.kt

```diff
@@ -105,6 +126,14 @@ internal class JsonLiteral internal constructor(
     }
 }
 
+/** Used as a marker to indicate the encoder should use `encodeInline()` */
+internal val jsonLiteralNumberDescriptor: SerialDescriptor = JsonLiteralNumber.serializer().descriptor
```

Comment:
1. ✅
2. `InlinePrimitiveDescriptor` is internal to kxs.core, so it can't be accessed in kxs.json. https://github.com/Kotlin/kotlinx.serialization/blob/daa95c79ffadc0eedbbb4a481a00556b78212e43/core/commonMain/src/kotlinx/serialization/internal/InlineClassDescriptor.kt#L28

Reply:
It worth changing to `@InternalSerializationApi public fun InlinePrimitiveDescriptor` then